cfgmgr/teammgr.cpp

```diff
@@ -594,6 +609,20 @@ task_process_status TeamMgr::addLagMember(const string &lag, const string &membe
     string res;
     uint16_t keyId = generateLacpKey(lag);
 
+    vector<FieldValueTuple> fvs;
+    m_cfgPortTable.get(member, fvs);
+
+    // Get the MACsec status
+    auto it = find_if(fvs.begin(), fvs.end(), [](const FieldValueTuple &fv) {
+            return fv.first == "macsec";
+            });
+    // Portchannel needs to be enabled after MACsec enabled
```

Comment:
Just curios, how is this handled if `macsec `is enabled after portchannel member is added?

Reply:
The MACsec will fail to enable if the port is a member of port channel. This solution isn't graceful so I'm investigating a better solution.